### ISSUE TITLE
[Batch Normal] Would you have plan to sync batch normalization?

### ISSUE THREAD

**Issue by yjl9122** [tensorflow/tensorflow#19412]
For semantic segmentation, it is very import to syncbn. would you tell us how to do it?

**Comment by ppwwyyxx** [comment]
Duplicate of #18222
